process/project-lifecycle.md

```diff
@@ -172,13 +172,23 @@ See [Submission Process](#submission-process) below and the [Archive application
 
 ### Project creation or change of lifecycle stage
 
-For initiating the creation of a new project or for requesting a change of a project's lifecycle stage, an application must be submitted to the TAC. To this end, a maintainer of the project (or anyone in the case of Archived projects) creates a PR in this repository with the following changes:
+For initiating the creation of a new initiative (project or SIG), an application must be submitted to an OpenSSF WG (or directly to the TAC if it wants to be a "top-level project" under the TAC). For submission to a WG, the project must be within scope of the WG. Of course, the WG must approve adding the initiative.
 
-* A new file in the `project-lifecyle-documents` directory containing all information requested for a project creation or a lifecycle change review. This file must be based on the template for the respective lifecycle stage in the `templates` directory. The `PROJECT_NAME_` prefix of the template must be replaced by the project name.
+If it's a pre-existing work (code, documentation, etc.), there are a few extra steps. We do *not* require copyright assignment, and future results will continue to be usable by its submitters. However, the submitters must be willing to license the work per the [OpenSSF Charter](https://openssf.org/about/charter/), that is, it must use a pre-approved license for its type or receive OpenSSF Governing Board approval. Pre-existing projects must notify the OpenSSF of any intellectual property of the project such as the existing code/documentation, trademarks, patents, and domain registrations. Please do this by sending an email to `operations` at `openssf.org`, so that we can perform a legal review. The licensing must be settled, trademarks and domains of the project itself must be reassigned to the OpenSSF, and patent issues must be dealt with before the project can join. This legal review should happen in parallel with other steps, to prevent slowing things down. Note: We generally expect that the current project leads are in favor of joining the OpenSSF. Even when it's legal to do so, we typically avoid forking whole projects, because our goal is to *collaborate* to produce great results.
+
+If a WG approves adding the project, the WG (usually a WG lead) must notify the TAC and give the TAC adequate time to object (e.g., at least 2 weeks). If TAC doesn't object, and the legal review approves, it's an OpenSSF project. Any TAC member may ask for further review, at which point it becomes a decision item for the TAC before it will be accepted by the OpenSSF.
```

Comment:
This paragraph is odd. Is it really saying you can have an OpenSSF project that is not yet accepted?

Reply:
No; this paragraph is about where to submit the OpenSSF project application documents to request either becoming an OpenSSF project, or if already an OpenSSF project, changing level (eg sandbox to graduated). Does that give it more context?

Reply:
Nope.

> If TAC doesn't object, and the legal review approves, it's an OpenSSF project.

Poof. It's an OpenSSF project.

> Any TAC member may ask for further review, at which point it becomes a decision item for the TAC before it will be accepted by the OpenSSF.

Ok, so it was an OpenSSF project, and then someone asked for further review? And now it might not be accepted?

This timeline doesn't make sense. The sentence order is almost certainly wrong.

Reply:
My apologies, I thought you were asking for clarification on the original text in line 175 and 177. I can't speak to the proposed changes.